app/components/FormFields/AvatarField.js

```diff
@@ -156,7 +191,22 @@ function AvatarField({ input, meta, disabled }) {
                 cropRadius={60}
                 closeIconColor="transparent"
                 onCrop={setY}
-                onBeforeFileLoad={() => {
+                label={
+                  isFileTooLarge ? (
+                    <FormattedMessage {...messages.fileSizeErrorMsg} />
+                  ) : (
+                    <FormattedMessage {...messages.chooseFile} />
+                  )
+                }
+                labelStyle={isFileTooLarge ? labelErrorStyle : {}}
+                onBeforeFileLoad={e => {
+                  const imgSizeInMb = e.target.files[0].size / 1000000;
+
+                  if (imgSizeInMb > IMG_SIZE_LIMIT) {
```

Comment:
If set value of IMG_SIZE_LIMIT in bytes then this logic does not need to divide the size. Size could be just compared with the constant.